utils/file.js

```diff
@@ -20,12 +20,29 @@ exports.fileSystem = (function(){
   const generatePrivateKey = () => {
     return crypto.randomBytes(32).toString('hex')
   }
-  const generateEnvFile = () => {
+  const generateEnvFile = (optionalVars) => {
+    let envVarsToWrite = '';
     if (!fileSystem.existsSync('./.env') || !dotenv.config().parsed.PRIVATE_KEY){
-      const privateKey = `PRIVATE_KEY=${generatePrivateKey()}`
-      fileSystem.writeFileSync('./.env', privateKey)
+      const privateKey = `PRIVATE_KEY=${generatePrivateKey()}\n`
+      envVarsToWrite = envVarsToWrite.concat(privateKey)
+    } 
+    if (!dotenv.config().parsed.STELLAR_ACCOUNT_ID || !dotenv.config().parsed.STELLAR_SECRET ){
```

Comment:
I know it's extra work, but it wouldn't be too hard to pull some of this into a helper function to clean up some of the logic.

```
hasExstingEnv() {
  return (!dotenv.config().parsed.STELLAR_ACCOUNT_ID || !dotenv.config().parsed.STELLAR_SECRET )
}
```

So you could then do this in the constructor

```
if (!hasExistingEnv) {
  let stellarKeyPair = stellar.generateKeys()
  fileUtils.generateEnvFile({'STELLAR_ACCOUNT_ID': stellarKeyPair.publicKey(), 
  'STELLAR_SECRET': stellarKeyPair.secret()})
}
```

Using this as a way to only run `generateEnv` when you need an env will allow you to remove the `if (envVarsToWrite !== '')` guard clause inside `generateEnv` as well.